css/easyzoom.css

```diff
@@ -46,6 +46,22 @@
 	background: #FFF;
 }
 
+.easyzoom-lens {
+	position: absolute;
+	top: 0;
+	left: 0;
+	background-color: #FFF;
+	background-color: rgba(255, 255, 255, 0.35);
+	border: 1px solid #CCC;
+	border-color: rgba(0, 0, 0, 0.35);
+	width: auto;
+	height: auto;
+	cursor: pointer;
```

Comment:
I think we should maintain the crosshair cursor, I think the pointer gives the wrong signals - IE. you can click when you can't.